scripts/apps/authoring/authoring/services/ConfirmDirtyService.js

```diff
@@ -64,6 +64,20 @@ export function ConfirmDirtyService($window, $q, $filter, api, modal, gettextCat
     };
 
     /**
+     * In case publish is triggered by quick buttons, show confirmation dialog
+     */
+    this.confirmQuickPublish = function(action) {
+        return modal.confirm(
+            $interpolate(gettextCatalog.getString(
+                'Do you want to publish the article?'
```

Comment:
What exactly are you interpolating here? There are no identifiers in any of these strings.